tests/acceptance/features/apiSpacesDavOperation/moveByFileId.feature

```diff
@@ -0,0 +1,154 @@
+Feature: moving/renaming file using file id
+  As a user
+  I want to be able to move or rename files using file id
+  So that I can manage my file system
+
+  Background:
+    Given using spaces DAV path
+    And user "Alice" has been created with default attributes and without skeleton files
+
+  Scenario Outline: move a file into a folder inside personal space
+    Given user "Alice" has created folder "/folder"
+    And user "Alice" has uploaded file with content "some data" to "/textfile.txt"
+    And we save it into "FILEID"
+    When user "Alice" moves a file "/textfile.txt" into "/folder" inside space "Personal" using file-id path "<dav-path>"
+    Then the HTTP status code should be "201"
+    And for user "Alice" the space "Personal" should contain these entries:
+      | folder/textfile.txt |
```

Comment:
Could you use this step instead while checking nested resources? Single item should be file.
I am working using default depth:infinity config so this is for early prevention
```suggestion
    And for user "Alice" folder "folder" of the space "Personal" should contain these entries:
      | textfile.txt |
```

Reply:
sure !!